modules/admin/src/views/layouts/nosession.php

```diff
@@ -12,6 +12,8 @@
             <?php $this->beginBody() ?>
                 <?= $content; ?>
             <?php $this->endBody() ?>
+            <? $this->registerJS('$(window).load(function () {checkInputLabels(); });'); ?>
```

Comment:
checkInputLabels() is already execute in the login.js script.

Reply:
yes, but we need to run this function in window on load context.

Reply:
we did, directly inside the login.js